hudi-gcp/src/main/java/org/apache/hudi/gcp/bigquery/BigQuerySyncTool.java

```diff
@@ -96,7 +96,7 @@ private void syncCoWTable(HoodieBigQuerySyncClient bqSyncClient) {
         .setUseFileListingFromMetadata(config.getBoolean(BIGQUERY_SYNC_USE_FILE_LISTING_FROM_METADATA))
         .setAssumeDatePartitioning(config.getBoolean(BIGQUERY_SYNC_ASSUME_DATE_PARTITIONING))
         .build();
-    manifestFileWriter.writeManifestFile();
+    manifestFileWriter.writeManifestFile(false);
 
```

Comment:
Wondering how the user can control this option?

Reply:
You mean to use the writer directly in your code then?

Reply:
Let's make it more general so that the SQL/DeltaStreamer user can also control this option, I'm wondering how necessary the change is, it is like a `must have` fix for GCP files?

Reply:
We can be conservertive to not explose the option to user first, can you rebase with the latest master and re-trigger the tests?